mlflow/pipelines/utils/step.py

```diff
@@ -66,6 +68,16 @@ def display_html(html_data: str = None, html_file_path: str = None) -> None:
         html_file_path = html_file_path if html_data is None else None
 
         if is_in_databricks_runtime():
+            from distutils.version import LooseVersion
+
+            dbr_version_image_key = get_databricks_runtime()
+            dbr_version = dbr_version_image_key.split("-")[0]
+            if LooseVersion(dbr_version).version[0].major < 11:
```

Comment:
Can we avoid using `distutils` because it's deprecated?

https://peps.python.org/pep-0632

```suggestion
            dbr_version_image_key = get_databricks_runtime()
            if int(dbr_version_image_key.split(".")[0]) < 11:
```

Reply:
Sounds good! Didn't realize `distutils` was deprecated.